templates/autoscale-new-vpc.template

```diff
@@ -958,6 +996,9 @@
                 "FortiAnalyzerVersion": {
                     "default": "FortiAnalyzer version"
                 },
+                "FortiAnalyzerCustomPrivateIpAddress": {
+                    "default": "FortiAnalyzer private IP"
```

Comment:
```suggestion
"default": "FortiAnalyzer private IP address"
```

Reply:
The spaces before "default" need to to remain. Could you please update your suggestion?